spec/factories.rb

```diff
@@ -131,10 +131,12 @@
   end
 
   factory :measure_condition do
```

Comment:
This was just broken and couldn't be used so I've taken an example measure condition returned from the api and recreated this based on what form that took